EIPS/eip-6358.md

```diff
@@ -12,37 +12,37 @@ created: 2023-01-17
 
 ## Motivation
 
-For projects serving multiple chains, it might be useful that the token is able to be accessed anywhere. Although assets-bridges can more or less make it, we don't think it is enough. And in the process of R&D, we found that the fragmentation of tokens is a common and disturbing problem among chains and L2s. 
+- The current paradigm of token bridges makes assets fragment.  
+- If ETH was transferred to another chain through the current token bridge, if the chain broke down, ETH will be lost for users.  
 
-- We want our token to be treated as a whole instead of being divided into fragmented parts on different public chains. O-DLT can get it.
-- When one chain breaks down, we don't want to lose our assets along with it. Assets-bridge paradigm cannot provide a guarantee for this. O-DLT can provide this guarantee even if there's only one chain that works, for example, we can rely on the stability and robustness of Ethereum.  
-- Not just for a certain token, we think the Omniverse Token might be useful for other projects on Ethereum and other chains. O-DLT is actually a new kind of open-source asset paradigm at the application level. 
+The core of this ERC is synchronization instead of transferring, even if all the other chains break down, as long as Ethereum is still running, user’s assets will not be lost. 
 
+- The fragment problem will be solved.
+- The security of users' multi-chain assets can be greatly enhanced.
 
 ## Specification
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
 
-*The word `Omniverse` in definitions will be substituted with the assigned number.*
-
 ### Omniverse Account
 
-The Omniverse account is RECOMMENDED to be expressed as a public key created by the elliptic curve `secp256k1`, which has already been supported by Ethereum tech stacks and can be used to generate an Ethereum address directly. For those who have a different address system, a mapping mechanism is RECOMMENDED.  
+There SHOULD be a global user identifier of this ERC, which is RECOMMENDED to be referred to as Omniverse Account (`o-account` for short) in this article.  
+The `o-account` is RECOMMENDED to be expressed as a public key created by the elliptic curve `secp256k1`. A mapping mechanism is RECOMMENDED for different environments.  
 
 ### Data Structure
 
-The definations of omniverse transaction data MUST be defined as follows:  
+A specific data structure SHOULD be designed to discribe the transactions need to be synchronized, and the name of the transactions of this ERC is RECOMMENDED to be referred to as Omniverse Transaction (`o-transaction` for short).  
+The data structure MUST be defined as follows:  
```

Comment:
You should omit the first bit (with "SHOULD") because immediately after you have a more specific "MUST" requirement.

```suggestion
An Omniverse Transaction (`o-transaction` for short) MUST be described with the following data structure:
```